cmake/modules/SwiftExternalBenchmarkBuild.cmake

```diff
@@ -74,6 +74,14 @@ function (add_external_benchmark_suite)
     list(APPEND all_stdlib_dependencies ${stdlib_dependencies})
   endforeach()
 
+  if(CMAKE_SYSTEM_NAME STREQAUAL Windows)
```

Comment:
This looks like a typo, I think you mean `STREQUAL`.

Reply:
Yeah, thats a typo :-(.  Will fix!